docs/app/pages/charts/charts-sections/organization-chart/organization-chart/organization-chart.component.html

```diff
@@ -87,6 +87,9 @@
     <tr uxd-api-property name="showReveal" type="boolean" defaultValue="false">
         Define if the reveal more button should be visible. This is used to display ancestor nodes, which may be initially hidden.
     </tr>
+    <tr uxd-api-property name="allowToggling" type="boolean" defaultValue="true">
+        Define if a node can be toggled or not.
```

Comment:
```suggestion
        Define if nodes can be toggled or not.
```
This should probably be changed to so it's clear that individual nodes can't have their toggling disabled, it's for all of them